src/applications/proxy-rewrite/README.md

```diff
@@ -1,80 +1,124 @@
+Previously, the VA.gov header/footer and the TeamSites header/footer shared quite a bit of code. This helped to reduce tech debt, code duplication and defects.
+
+However, an effort is underway to convert the VA.gov header/footer to web components in the Design System library. Because the code between the VA.gov header/footer and TeamSites header/footer is shared **and** TeamSites do not support web components, we needed to separate the code for Teamsites as much as possible.
+
+As a result, all of the TeamSites header/footer code is now contained in `src/applications/proxy-rewrite` and is 100% duplicative of the VA.gov header/footer. There is a single style file in `proxy-rewrite/sass`, and all of the injection code is still present within this folder. The application has very little dependency on other code within the vets-website repo (with the exception of Platform utilities).
+
+### The downsides of an encapsulated injected header/footer app
+Aside from tech debt and code duplication:
+
+1. Any visual or structural changes made to the VA.gov (modernized) header/footer should also be applied and tested in the injected header/footer to keep the experience consistent
+2. The injected header/footer uses some Platform utility functions. If there are changes to these utilities, you must remember to test those changes against the injected header/footer
+
+## Do I need content-build running locally to test TeamSites?
+**No**. TeamSites do not use `content-build` as they do not use Drupal and thus do not have statically-generated Drupal pages. However, `content-build` generates a static JSON file with all of the Drupal data for the header/footer. You will need a `headerFooter.json` file in `vets-website/build/localhost/generated/`. 
+
+This file is created by running `yarn build` from `content-build`. Without optimization, this build step can take upwards of 8 hours. [Refer to this guide](https://github.com/department-of-veterans-affairs/content-build?tab=readme-ov-file#optimizing-build-time) to optimize your `content-build` and more quickly get up and running. **You will only need to do this once**, or whenever Drupal data for the header/footer is updated and affects your testing. Otherwise, as long as the `headerFooter.json` file exists, you can test the injected header/footer locally.
+
+Note: [next-build](https://github.com/department-of-veterans-affairs/next-build) which will eventually replace `content-build` in its entirety but will pull Drupal data the same way.
+
+## Where does the header/footer data come from?
+The header/footer data is largely populated by Drupal. When `content-build` runs its build, it fetches the Drupal data (outlined below) and uses [create-header-footer.json](https://github.com/department-of-veterans-affairs/content-build/blob/main/src/site/stages/build/plugins/create-header-footer.js) to structure the data object and generate the `headerFooter.json` inside the `vets-website` build/generated folder.
```

Comment:
@chriskim2311 Originally this line only had the first sentence. I expanded on it a bit to include info about how `content-build` populates the data for `vets-website`.